crmsh/bootstrap.py

```diff
@@ -186,6 +189,20 @@ def init_sbd_manager(self):
         from .sbd import SBDManager
         self.sbd_manager = SBDManager(self)
 
+    def load_profiles(self):
+        """
+        Load profiles data for different cloud provider
+        """
+        env_name = utils.detect_cloud()
+        if not env_name:
+            return
```

Comment:
In stead of directly "return". I suggest to set

env_name = "user_customized"

This makes the log more generic and gives users the opportunity to define the "user_customized" profile in "profiles.yaml", for example, when their environment is on-premise.

Reply:
Changed